### DISCUSSION TITLE
How to specify data from a specific project in grafana?

### DISCUSSION THREAD

**Discussion by pedraamn** [apache/devlake#6299] [Q&A] [answered]
Hi. When I port-forward to grafana pod it has these general for github and what not like this picture
<img width="1721" alt="image" src="https://github.com/apache/incubator-devlake/assets/77109389/b53ed251-f407-448a-8b42-4aee9f7165be">

Even inside some of the dashboards, like github, you can check specific repositorys as sources. But how can I use a specific project as the source? Let's say I want to see JIRA data from project 1 or I want to see github data from project 2, is there a way to set these filters up?

**Comment by klesh** [comment, marked as answer]
@pedraamn I don't think it is available in the prebuilt dashboards, but you can always craft your own dashboard and SQL to achieve what you want or need.
